### ISSUE TITLE
PSI version scores differs from built-in chrome Lighthouse tool

### ISSUE THREAD

**Issue by memegauste** [GoogleChromeLabs/psi#127]
Hiya, so I assume PSI in-hood engine is the same as Lighthouse, but it's not.
I expect the same scores in both and I get ~5 points difference.
It's not that bad until you imagine that 87 points instead of 92 gives you orange color in benchmark.

Is there way to get more consistent scores with less point margin (like 1-2% difference)?

**Comment by vanushwashere** [comment]
Yeah, it gives me a 10+ points difference with the [web.dev](https://pagespeed.web.dev/?utm_source=psi&utm_medium=redirect) version as well.

**Comment by charkour** [comment]
Are you running lighthouse through your web browser? If so, then it is running in a different environment than PSI online. You'll want to match the environments when you run tests. 

![image](https://user-images.githubusercontent.com/33156025/180480928-e4245955-3edf-43ca-bf93-18cc93d7bc0f.png)

I hope this helps!
